README.md

```diff
@@ -190,6 +190,21 @@ Sent from the rover server to inform Mission Control of a joint's current positi
 - `joint` - the name of the joint
 - `position` - the current position in degrees
 
+## Set Inverse Kinematics
```

Comment:
This should probably be called Set Arm IK Enabled, since this just enables or disables it.